mkdocs.yml

```diff
@@ -57,6 +57,7 @@ nav:
     - 'Code of Conduct': './process/code_of_conduct.md'
 - Activity Status:
     - 'Proposed': './results/proposed.md'
+    - 'Governance Stack Design Principles': './results/DP0007-governance-stack.md'
```

Comment:
This change is in error. The Yml ios not suppose to be changed. Only the phased files (proposed, accepted, approved). In this case `proposed` already has an entry for DP0007.

Reply:
Sorry, my bad. I didn't plan to the file in the repo. Have forgotten to remove before merge. It should be deleted.